### ISSUE TITLE
A high-level explainer of IPLD.

### ISSUE THREAD

**Issue by Mr0grog** [ipfs/ipfs-docs#242]
## This issue has a bounty!
Successfully closing this issue by producing a production-ready piece of documentation can earn you not only the undying love of the IPFS community — it can net you a financial reward. [See the current list of bounty issues and their values here.](https://github.com/ipfs/devgrants/projects/1)

## The need in brief
A high-level explainer of IPLD.

Context/supporting materials: At the IPFS developer summit in Berlin in July 2018, we had poster-making sessions where people explored various IPFS concepts. This IPLD poster could ideally be expanded into a basic diagram of what IPLD is and how it works, supplemented by written content.

![ipld](https://user-images.githubusercontent.com/74178/43231007-6e49612a-901f-11e8-947d-9bef93f2ea44.jpg)

## Deliverable
A concise, high-level description of this IPFS concept. It should answer the following, **supplemented whenever possible by diagrams, videos and/or code examples**:

- **What** is this?
- How does it **relate** to the rest of IPFS?
- What are some **generic use cases** for this — how might it meet a need in someone's project/goal?
- How can (or should?) you **use** it? (Note: sometimes the answer is that you should understand it, but *not necessarily* use it based on your use case or the current state of the tech.)
- **Where** do you go to learn more?
- What is the **current state** of this concept or technology?

**Content should take the form of a Markdown document, supplemented by diagrams as embedded images.** File locations should be as follows:
- MD content should live here (lease make a PR against the existing placeholder document) https://github.com/ipfs/ipfs-docs-v2/blob/master/docs/concepts/ipld.md
- Images can live in the MD content's parent directory; we'll move them before final merge
- For videos, just post temporarily on the platform of your choice, and we'll sort out on a case-by-case basis

## Guidelines
- Please use the [IPFS Docs Contributor Guidelines](https://github.com/ipfs/docs/blob/master/CONTRIBUTING.md ) as your north star — **adherence to these guidelines are crucial when it comes to awarding potential bounty**!
- Also, please keep in mind our [core user personae](https://app.mural.co/t/protocollabs6957/m/protocollabs6957/1564779785852/cf7669f3c1773508a811a3fa0eadfb99efb310bf). If a piece of content serves one or more personae in particular, make sure it checks out with that persona's attributes.

**Comment by realChainLife** [comment]
@johnnymatthews is anyone still assigned to this?

**Comment by johnnymatthews** [comment]
Thanks for your interest here @realChainLife, but we're waiting on some internal changes to IPLD before releasing this issue. It's not available for public contributions just yet.

**Comment by bibek54** [comment]
Any idea on how long will it take to get it published?

**Comment by johnnymatthews** [comment]
This item is set to low priority right now. I don't have a timeframe right now, but it will likely get addressed early next quarter.

If you feel this issue should have a higher priority, please feel free to make that case here. It's currently set as `P3` not because it's unimportant, but because there are more pressing issues right now.

**Comment by ChrisHagglundWS** [comment]
not like its too hard to just pull cids out of javascript data structure and then go fetch the content of those as needed all the way down. this seems like more of a concept than a spec

**Comment by ROBERT-MCDOWELL** [comment]
is there any updated roadmap on IPLD?

**Comment by BlocksOnAChain** [comment]
https://ipld.io/docs/intro/primer/ - closing since we have this website which is explaining everything in detail.

**Comment by Martinligabue** [comment]
the page still links here, should it be updated?
https://docs.ipfs.io/concepts/ipld/

**Comment by noryev** [comment]
https://ipld.io/docs/ 
I think this is where the link should take us.

**Comment by johnnymatthews** [comment]
Thanks for the reminder! I've changed the IPLD link to go to the ipld.io/docs site in https://github.com/ipfs/ipfs-docs/pull/1077.
